### PR TITLE
Milestone 2: Add, update and delete features

### PR DESCRIPTION
**Pull request by carloshs1994**
# in this Milestone:

✅ Implement a function for adding a new task (add a new element to the array).
✅ Implement a function for deleting a task (remove an element from the array).
✅ Implement a function for editing task descriptions.
✅ By default new tasks have the property `completed` set to `false` and the property `index` set to the value of the new array length
✅ Deleting a task update all remaining items' indexes, so they represent the current list order and are unique.
✅ All changes to the To-Do List are saved in local storage.

### CODE REVIEW

File: src/add-and-remove.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('next milestone');
```
Comment (MrBrN197) [inline src/add-and-remove.js]: - [x] Hi, Please ensure to place the functionality for adding and removing & editing into this add-and-remove.js file. please remove the code from index.js and place the new functionality here.

File: src/index.js [hunk 2]
```diff
@@ -60,8 +42,63 @@ const createTask = (input) => {
...
-};
+  document.querySelectorAll('.hello').forEach((e) => {
+    e.addEventListener('keyup', (event) => {
+      listOfTasks[event.target.id].description = event.target.value;
+    });
+  });
+  document.querySelectorAll('li.hello > img').forEach((task) => {
+    task.addEventListener('click', (event) => {
+      if (event.target.src === Trash) {
+        // eslint-disable-next-line no-use-before-define
```
Comment (MrBrN197) [inline src/index.js]: - [x] Hi, your code is working fine but kindly try to fix this ESlint error. this is usually a warning you have written some code that is not clean. please address this issue.

File: src/index.js [hunk 3]
```diff
@@ -1,56 +1,38 @@
...
     const li = document.createElement('li');
-    // Lodash, now imported by this script
     li.innerHTML = `
       <input type="checkbox" id="task" name="task-${task.index + 1}">
-      <label for="task-${task.index + 1}">${task.description}</label>
-      <img src="${DotMenu}" alt="Delete or Drag and drop">
+      <input type="text" value="${task.description}" id="${task.index}" class="task-text">
+      <img class="${task.index}" src="${DotMenu}" alt="Delete or Drag and drop">
     `;
     li.classList.add('hello');
```
Comment (MrBrN197) [inline src/index.js]: - [ ] [OPTIONAL] Please rename this class from 'hello' to something that has some semantic significance. the name of the class should usually describe or have a relationship to how it augments the element itself.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @carloshs1994  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔ JavaScript functionality works correctly.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @carloshs1994, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**